pkg/cmd/cli/cmd/importimage.go

```diff
@@ -249,151 +249,134 @@ func (o *ImportImageOptions) waitForImport(resourceVersion string) (*imageapi.Im
 }
 
 func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imageapi.ImageStreamImport, error) {
+	var isi *imageapi.ImageStreamImport
 	stream, err := o.isClient.Get(o.Name)
-	from := o.From
-	tag := o.Tag
-
+	// no stream, try creating one
 	if err != nil {
 		if !errors.IsNotFound(err) {
 			return nil, nil, err
 		}
 		if !o.Confirm {
```

Comment:
Let's remove this check - it's annoying to end users and doesn't really protect you from anything (if you typo, you can just delete it)